google-datacatalog-connectors-commons/src/google/datacatalog_connectors/commons/config/yaml_config.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import yaml
+
+
+class YamlConfig:
+
+    @classmethod
+    def parse_as_dict(cls, content):
+        return yaml.load(content, Loader=yaml.FullLoader) or dict()
```

Comment:
In which scenario an empty `dict` is returned? Sorry, I didn't get it from the test cases...

Reply:
I verified and I wasn't able to simulate that... so removed the empty `dict` return.